src/components/map/map.tsx

```diff
@@ -169,6 +223,50 @@ export class Map implements GxComponent {
     }
   }
 
+  private updateSelectionMarkerPosition() {
+    const centerCoords = this.map.getCenter();
+    this.centerCoords = `${centerCoords.lat},${centerCoords.lng}`;
+    this.selectionMarker.setAttribute("coords", this.centerCoords);
```

Comment:
We could declare `selectionMarker` with a more specificc type, taking advantage of `HTMLGxMapMarkerElement` (is a type automatically generated by Stencil in `components.d.ts`).
That way, we could set the `coords` property, instead of `setAttribute`:

```
this.selectionMarker.coords = this.centerCoords;
````

Reply:
Oh! So convenient!
I had a lot of obstacles with that problem... `selectionMarker: Element` was the final conclusion
I will try your solution! Thx!

Reply:
Mmm... There is a conflict here.

If I make `selectionMarker: HTMLGxMapMarkerElement` there will be a type error when I search for slot with `this.element.querySelector("[slot='selection-layer-marker']")`
The querySelector returns an `Element` type 😣

![image](https://user-images.githubusercontent.com/33690842/83319748-7e971700-a217-11ea-9cfb-176c3d4cdb94.png)

Reply:
Try this: 

`const slot = this.element.querySelector<HTMLGxMapMarkerElement>("[slot='selection-layer-marker']")`

Reply:
😮 
i didn't know that. 
I saw it in  the popup dialog, but I did not want to try...
I will try! 😄

Reply:
It works! The latest tests have passed successfully with these new changes.
See new changes in ad11b7e